Ticket - 29549 Customization to reports with RFC logo.

sewa67 opened the pull request:
Please review this change and merge into development branch.
Files - arrep/r-stmt.w, oe/rep/bolrfcx.i, oe/rep/relacpi,i, oe/rep/relacpi2.i, 
 cec/quote/quorfc2.i, cec/quote/quorfc.p, cec/quote/quorfc.i, sys/ref/sys-ctrl.i, est/r-quoprt.w

sewa67 committed:
Ticket - 29549 Customization to reports with RFC logo.
Files - arrep/r-stmt.w, cec/quote/quoxpnt2.i, cec/quote/quoxprnt.p, oe/rep/bolrfcx.i, oe/rep/relacpi,i, oe/rep/relacpi2.i, po/po-xprnt.i,
po/po-xprnt.p

sewa67 committed:
Ticket - 29549 Customization to reports with RFC logo.
Files - arrep/r-stmt.w, oe/rep/bolrfcx.i, oe/rep/relacpi,i, oe/rep/relacpi2.i,
 cec/quote/quorfc2.i, cec/quote/quorfc.p, cec/quote/quorfc.i, sys/ref/sys-ctrl.i, est/r-quoprt.w

brad-advantzware reviewed (changes requested):
Need new RFC format for RELPRINT.

sewa67 committed:
Ticket - 29549 Customization to reports with RFC logo
Files - oe/rep/bolrfcx.i, oe/rep/bolrfcx2.i, oe/rep/bolrfcx3.i, oe/rep/bolrfcxcons.i, oe/rep/bolrfcxcons2.i, cec/quote/quorfc.p, cec/quote/quorfc2.i,   cec/quote/quorfc.i,  oerep/r-relprt.w, oe/rep/relrfc.p, oe/rep/relrfc.i, oe/rep/relrfc2.i,arrep/r-stmt.w,sys/ref/sys-ctrl.i

Commit:
Merge branch 'develop' into feature/29549_Customization_to_reports_with_RFC_logo
